src/probnum/filtsmooth/gaussfiltsmooth/unscentedkalman.py

```diff
@@ -264,25 +264,7 @@ def from_ode(
         prior,
         evlvar=0.0,
     ):
-
-        spatialdim = prior.spatialdim
-        h0 = prior.proj2coord(coord=0)
-        h1 = prior.proj2coord(coord=1)
-
-        def dyna(t, x):
-            return h1 @ x - ode.rhs(t, h0 @ x)
-
-        def diff(t):
-            return evlvar * np.eye(spatialdim)
-
-        def diff_cholesky(t):
-            return np.sqrt(evlvar) * np.eye(spatialdim)
-
-        disc_model = pnss.DiscreteGaussian(
-            input_dim=prior.dimension,
-            output_dim=prior.spatialdim,
-            state_trans_fun=dyna,
-            proc_noise_cov_mat_fun=diff,
-            proc_noise_cov_cholesky_fun=diff_cholesky,
+        discrete_model = pnss.DiscreteGaussian.from_ode(
+            ode=ode, prior=prior, evlvar=evlvar
         )
-        return cls(disc_model)
+        return cls(discrete_model)
```

Comment:
Just a minor thing: Do you think it sensible to outsource this in a similar way in the `EKF`? With an optional `jacobian` kwarg and a EK0 or EK1 check in the `DiscreteEKFComponent.from_ode` method?

Reply:
Do you mean in the sense of `from_ode(ivp, prior, evlvar=0., ek0_jacobian=False)` that essentially moves the `ek0_or_ek1` decision up one level?

Reply:
Hmmmm I am not so sure behind the logic here. I see what you're trying to do, but there should be a better way. For this PR, I'd propose to leave it as is and refactor this. (I am thinking #361)